packages/core/src/watch.ts

```diff
@@ -123,9 +124,11 @@ export const submitPatches = (
 
   logger.info("[update cycle]", comps);
 
+  event.rerender = {};
   Object.keys(comps).forEach(compId => {
     const { setState, name, newValues, status } = comps[compId];
     if (!status.unmounted) {
+      event.rerender![comps[compId].name] = true;
```

Comment:
Why is this needed?

Reply:
so that we can log this to the console, and show which components are re-rendered as a result of a state update